# Global Expansion of SARS-CoV-2 Variants of Concern: Dispersal Patterns and Influence of Air Travel

Houriiyah Tegally<sup>1,2§</sup>, Eduan Wilkinson<sup>1§</sup>, Darren Martin<sup>3,4</sup>, Monika Moir<sup>1</sup>, Anderson Brito<sup>5</sup>, Marta Giovanetti<sup>6,7,8</sup>, Kamran Khan<sup>9,10</sup>, Carmen Huber<sup>9</sup>, Isaac I. Bogoch<sup>10</sup>, James Emmanuel San<sup>2</sup>, Joseph L.-H. Tsui<sup>14</sup>, Jenicca Poongavanan<sup>1</sup>, Joicymara S. Xavier<sup>1,11,17</sup>, Darlan da S. Candido<sup>12</sup>, Filipe Romero<sup>12</sup>, Cheryl Baxter<sup>1</sup>, Oliver G. Pybus<sup>14,15,18</sup>, Richard Lessells<sup>2</sup>, Nuno R. Faria<sup>12,13,14</sup>, Moritz U.G. Kraemer<sup>14,15*</sup>, Tulio de Oliveira<sup>1,2,16*</sup>

Affiliations

1Centre for Epidemic Response and Innovation (CERI), School of Data Science and Computational Thinking, Stellenbosch University, Stellenbosch, South Africa

2KwaZulu-Natal Research Innovation and Sequencing Platform (KRISP), Nelson R. Mandela School of Medicine, University of KwaZulu-Natal, Durban, South Africa

3Wellcome Centre for Infectious Diseases Research in Africa (CIDRI-Africa), Cape Town, South Africa

4Institute of Infectious Disease and Molecular Medicine, University of Cape Town, Cape Town, South Africa

5Instituto Todos pela Saúde, São Paulo, São Paulo, Brazil

6Laboratorio de Flavivirus, Fundacao Oswaldo Cruz, Rio de Janeiro, Brazil

7Laboratório de Genética Celular e Molecular, Universidade Federal de Minas Gerais, Belo Horizonte, Brazil

8Department of Science and Technology for Humans and the Environment, University of  Campus Bio-Medico di Roma, Rome, Italy.

9BlueDot, Toronto, Canada

10Department of Medicine, Division of Infectious Diseases, University of Toronto, Toronto, Canada

11Universidade Federal de Minas Gerais, Belo Horizonte, Brazil

12MRC Centre for Global Infectious Disease Analysis and Department of Infectious Disease Epidemiology, Jameel Institute, School of Public Health, Imperial College London, London, UK

13Departamento de Moléstias Infecciosas e Parasitárias e Instituto de Medicina Tropical da Faculdade de Medicina, Universidade de São Paulo, São Paulo, Brazil
14Department of Biology, University of Oxford, Oxford, UK

15Pandemic Sciences Institute, University of Oxford, Oxford, UK

16Department of Global Health, University of Washington, Seattle, WA, USA

17Institute of Agricultural Sciences, Universidade Federal dos Vales do Jequitinhonha e Mucuri, Unaí, Brazil

18Department of Pathobiology and Population Sciences, Royal Veterinary College London, London, UK

§These authors contributed equally
*These authors jointly supervised this work
Correspondence: houriiyah.tegally@gmail.com, tulio@sun.ac.za  and moritz.kraemer@biology.ox.ac.uk 


## Summary

In many regions of the world, the Alpha, Beta and Gamma SARS-CoV-2 Variants of Concern (VOCs) co-circulated during 2020-21 and fueled waves of infections. During 2021, these variants were almost completely displaced by the Delta variant, causing a third wave of infections worldwide. This phenomenon of global viral lineage displacement was observed again in late 2021, when the Omicron variant disseminated globally. In this study, we use phylogenetic and phylogeographic methods to reconstruct the dispersal patterns of SARS-CoV-2 VOCs worldwide. We find that the source-sink dynamics of SARS-CoV-2 varied substantially by VOC, and identify countries that acted as global hubs of variant dissemination, while other countries became regional contributors to the export of specific variants. We demonstrate a declining role of presumed origin countries of VOCs to their global dispersal: we estimate that India contributed <15% of all global exports of Delta to other countries and South Africa <1-2% of all global Omicron exports globally. We further estimate that >80 countries had received introductions of Omicron BA.1 100 days after its inferred date of emergence, compared to just over 25 countries for the Alpha variant. This increased speed of global dissemination was associated with a rebound in air travel volume prior to Omicron emergence in addition to the higher transmissibility of Omicron relative to Alpha. Our study highlights the importance of global and regional hubs in VOC dispersal, and the speed at which highly transmissible variants disseminate through these hubs, even before their detection and characterization through genomic surveillance.

## Notes on Data and Code:
1. The findings of this study are based on sequences and metadata associated with a total of XXX sequences available on GISAID up to November 19, 2022, via gisaid.org/EPI_SET_XXX. Custom data sources and scripts to reproduce the results of this study are publicly shared in this repository. The repository contains all of the time scaled ML tree topologies, annotated tree topologies as well as custom data analysis and visualization scripts. Other datasets and pipelines used in this study are openly available and described in the Materials and Methods section. 

2. Sequences and Metadata associated with GISAID entries are not directly provided in this repository and should rather be downloaded directly from the GISAID database (gisaid.org)

3. The files in ViralTransitions folder are the outputs of the Phylogenetic Reconstruction workflow (described in Materials and Methods of the paper) and provided as input of downstream data analysis and visualization R scripts in FigureScipts

4. The intermediate files and tree annotation scripts are provided in the PhylogeneticReconstruction folder.

5. Raw air passenger travel data is subject to licensing agreements and provided in the Data folder as aggregated data files instead (aggregated by time-period or location).


>>>>>>> 82cb3ef5659a1b6fb556562e23ba0d54b15a1443
